managed/services/management/agent_test.go

```diff
@@ -113,32 +116,23 @@ func TestAgentService(t *testing.T) {
 			pmmAgent, err := models.CreatePMMAgent(s.db.Querier, models.PMMServerNodeID, nil)
 			require.NoError(t, err)
 
-			mysqldExporter, err := models.CreateAgent(s.db.Querier, models.MySQLdExporterType, &models.CreateAgentParams{
-				PMMAgentID: pmmAgent.AgentID,
-				ServiceID:  service.ServiceID,
-				Password:   "password",
-				Username:   "username",
-			})
-			require.NoError(t, err)
-
```

Comment:
I think it's better to keep it as close to reality as possible and in real cases we usually add mysqld_exporter

Reply:
Absolutely, will take care of it on the next iteration!